.sync.yml

```diff
@@ -1,6 +1,12 @@
 ---
 Rakefile:
   unmanaged: true
+Gemfile:
+  optional:
+    ':system_tests':
+      - gem: beaker-testmode_switcher
+        version: '<= 0.4.0'
```

Comment:
This sets a maximum version of `0.4.0`... but we want a minimum of `0.4.0`. Any of these would work:

* `~> 0.4` - pessimistically versioned to `< 1.0`
* `= 0.4.0` - pin
* `>= 0.4.0` - anything going forward

First option is probably the safe bet for now...